### PR TITLE
Symfony3 upgrade

### PR DESCRIPTION
**Pull request by pkruithof**
* Symfony3 support
* Made services private
* Removed Behat test for configuration: the same thing was already tested via a unit test

### CODE REVIEW

**Comment by cleentfaar** [comment]
We could write some more tests for classes in the DI namespace, created an issue for it: https://github.com/treehouselabs/queue-bundle/issues/23

**Comment by cleentfaar** [comment]
lgtm

File: composer.json [hunk 1]
```diff
@@ -10,25 +10,29 @@
...
-    "phpunit/phpunit":              "~4.0",
-    "behat/behat":                  "~3.0",
-    "symfony/proxy-manager-bridge": "~2.4",
-    "ocramius/proxy-manager":       "~0.5",
-    "empi89/php-amqp-stubs":        "dev-master",
-    "behat/symfony2-extension":     "~2.0"
+    "ext-amqp":                     "^1.7",
+    "doctrine/orm":                 "^2.3",
+    "phpunit/phpunit":              "^4.0",
+    "behat/behat":                  "dev-master as 3.0",
```
Comment (pkruithof) [inline composer.json]: This is temporary, until behat/behat#806 is closed (new release tagged).

**Comment by fieg** [comment]
lgtm

**Pull request merged**